### ISSUE TITLE
Implement nnUNet example based on MONAI workflows

### ISSUE THREAD

**Issue by Nic-Ma** [Project-MONAI/MONAI#976]
**Is your feature request related to a problem? Please describe.**
As JoC team already built wonderful nnUNet example based on MONAI modules and PyTorch Lightning, it would be better to use MONAI workflows and make an example in MONAI repo.

**Comment by tvercaut** [comment]
This is related to #246

**Comment by Nic-Ma** [comment]
Addressed this ticket by PR: https://github.com/Project-MONAI/Tutorials/pull/20
Thanks.

**Comment by tvercaut** [comment]
This might be best span out as a seprate issue but as discussed by @FabianIsensee in #1013, "nnU-Net is so much more than the architecture". I fear my might be introducing confusion in the way we are referring to nnU-net currently within MONAI. Unless we have the automated configuration of the pipeline reimplemented, maybe we should use another terminology? Maybe dynU-Net? @FabianIsensee any thoughts?

I am also copying @martaranzini and @LucasFidon as they should soon be able to share some interesting experience comparing the nnU-Net code base with MONAI (related to #807).

**Comment by FabianIsensee** [comment]
Hi, 
dyn_UNet or similar is a good choice I think. I am also curious about the results of @martaranzini. Are there any updates?
Best,
Fabian

**Comment by martaranzini** [comment]
Hi Fabian,
Yes as @tvercaut mentioned, we do have quite some interesting results comparing MONAI and nnU-Net. In short, nnU-Net did not show the training instability we observed with MONAI, but we also noticed some substantial differences in the Dice loss implementation that we investigated deeper to compare their performance. 
We are now putting together the final analysis and we aim to report back to issue #807 hopefully by the beginning of next week.
Apologies for the delay in getting back, we had some issues with our machines while running the experiments.

Best,
Marta
